scripts/ci/determine-tests.mjs

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env node
+import {setOutput} from '@actions/core';
+import {readdirSync, statSync} from 'fs';
+import {EOL} from 'os';
+import {basename, dirname, join, relative} from 'path';
+import {getBaseHeadSHAs, getChangedFiles} from './hasFileChanged.mjs';
+import {listImports, ensureFileExists} from './list-imports.mjs';
+
+class NoRelevantChangesError extends Error {
+  constructor() {
+    super('Change detected elsewhere, running no tests.');
+    this.name = 'NoRelevantChangesError';
+  }
+}
+
+class DependentPackageChangeError extends Error {
+  constructor(file) {
+    super(
+      `Change detected in an atomic dependent Coveo package: ${file}. Running all tests.`
```

Comment:
```suggestion
      `Changes detected in a package on which Atomic and Quantic depend: ${file}. Running all tests.`
```

Reply:
Only atomic. This action determines the atomic tests only. Ill make the string better thank you.